app/assets/stylesheets/application_rmc-legacy.scss

```diff
@@ -0,0 +1,40 @@
+@import 'base';
+
+// Theme designed to include a modified version of the original RMC masthead image in larger screen sizes
+
+.masthead {
+  .site-title-container {   
+    .site-title-wrapper {
+      // Remove block background behind text
+      background: none;
+    }
+
+    @media (min-width: breakpoint-max('md')) {
+      // For larger screen sizes, hide title text (except for screen readers)
+      height: 178px;
+      .site-title {
+        @include sr-only;
+        }
+    }
+  }
+
+  .background-container {
+    // Remove blur filter and center masthead image
+    filter: none;
+    background-position: center;
+    height: 178px;
+
+    @media (max-width: breakpoint-max('md')) {
+      // For smaller screen sizes, zoom in on solid background color on left side of modified RMC masthead image
+      background-position: left;
+      background-size: 600% 100%; 
+    }
+  }  
+
+  .background-container-gradient {
```

Comment:
for smaller screens, this gradient is applied and the background is stretched  by width so it focuses on the far left part of the background which I think works, but I'm wondering about the colors of the title and the background image... will the title always have enough contrast to be readable and pass accessibility tests? Do any of the legacy sites have black masthead... I think that is why there is a default block used in the default theme, to take into account the various colors that could be used in the background image.

Reply:
Good point. I put the text block back in for the mobile view.